### PR TITLE
Fix "no schema found" error message

### PR DESCRIPTION
**Pull request by martinbonnin**
Before that the error message was this:

```
"Schema(s) found: But none of them contain type definitions."
```

### CODE REVIEW

File: libraries/apollo-compiler/src/main/kotlin/com/apollographql/apollo3/compiler/ApolloCompiler.kt [hunk 1]
```diff
@@ -60,6 +60,10 @@ object ApolloCompiler {
       it.toSchemaGQLDocument()
     }
 
+    if (schemaDocuments.isEmpty()) {
+      error("No schema found. Apollo needs a `.graphqls` or a `.json` schema.")
```
Comment (BoD) [inline libraries/apollo-compiler/src/main/kotlin/com/apollographql/apollo3/compiler/ApolloCompiler.kt:64]:
```suggestion
      error("No schemas found. Apollo needs a `.graphqls` or a `.json` schema.")
```

But we should confirm with a native english speaker to be sure 😅

**Review by BoD — APPROVED** [review]
👍

File: libraries/apollo-compiler/src/main/kotlin/com/apollographql/apollo3/compiler/ApolloCompiler.kt (hunk 1 above)
Comment (martinbonnin) [reply libraries/apollo-compiler/src/main/kotlin/com/apollographql/apollo3/compiler/ApolloCompiler.kt:64]: Ping @Meschreiber. Do you know what's the correct english there?

Comment (Meschreiber) [reply libraries/apollo-compiler/src/main/kotlin/com/apollographql/apollo3/compiler/ApolloCompiler.kt:64]: @martinbonnin , either could be correct depending on the context. If we're looking for multiple schemas and didn't find any, we can stick with "No schemas found."

Comment (martinbonnin) [reply libraries/apollo-compiler/src/main/kotlin/com/apollographql/apollo3/compiler/ApolloCompiler.kt:64]: We're looking for only 1 "main" schema. There could other "extra.graphqls" files but most often, they should only contain type extensions, not type definitions.

Comment (Meschreiber) [reply libraries/apollo-compiler/src/main/kotlin/com/apollographql/apollo3/compiler/ApolloCompiler.kt:64]: Got it. If we only care about finding the one main schema, "No schema found." is more accurate.

Comment (martinbonnin) [reply libraries/apollo-compiler/src/main/kotlin/com/apollographql/apollo3/compiler/ApolloCompiler.kt:64]: Thank you 🙏 . @BoD I'll stick with the current version but I'm happy to discuss it with Carm over risotto 😄

Comment (BoD) [reply libraries/apollo-compiler/src/main/kotlin/com/apollographql/apollo3/compiler/ApolloCompiler.kt:64]: Hahaha, with pleasure! 😂

**Pull request merged**